docker/release_dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:16.04
+
+ENV IROHA_HOME /opt/iroha
+ENV IROHA_BUILD /opt/iroha/build
+
+COPY iroha.deb /tmp/iroha.deb
+RUN apt-get update
+
+RUN apt-get install libc-ares-dev -y
+
+#Install iroha
+RUN dpkg -i /tmp/iroha.deb || true //continue executing even if command fails
+
+#Install iroha dependecies
+RUN apt-get install -y -f
+
+#Insall iroha again
+RUN dpkg -i /tmp/iroha.deb
```

Comment:
1. new line missing
2. afaik after fixing deps via apt-get there's no need to run installation again, isn't it?

Reply:
first you try to dpkg install, it says that it can't install iroha because of missing dependencies.
then you write `apt install -f`, it installs all deps.
then you install with dpkg again.

Reply:
Look at [this](https://unix.stackexchange.com/a/159103)